.codebuild/buildspec.deploy.demo.yml

```diff
@@ -0,0 +1,23 @@
+# buildspec for deploying latest changes to main development branch
+version: 0.2
+phases:
+  install:
+    commands:
+      - echo install step...
+  pre_build:
+    commands:
+      - echo logging in to AWS ECR...
+      - $(aws ecr get-login --no-include-email --region us-east-1)
+  build:
+    commands:
+      - echo build Docker image on `date`
+      - docker build -f Dockerfile.prod -t pwa-demo:latest .
+      - docker tag pwa-demo:latest 276375911640.dkr.ecr.us-east-1.amazonaws.com/pwa-demo
```

Comment:
PR review Ron: add second tag with a version - deploy latest still - use semantic versioning (version file) to keep track of which version is deployed and which was the last deployed.